src/create-reminder-signup/lib/identity.ts

```diff
@@ -36,13 +41,75 @@ export const getIdentityIdByEmail = async (
 	}
 
 	return response.json().then((identityResponse) => {
-		if (identityResponse.user?.id) {
+		if (
+			identityResponse &&
+			identityResponse.user &&
+			identityResponse.user.id
```

Comment:
:(

Reply:
yes :(
For the record - we couldn't get optional chaining to work with esnext.
The solution probably involves babel (see dcr) but it's probably not worth it here